### DISCUSSION TITLE
Login issues

### DISCUSSION THREAD

**Discussion by WadeRolfe** [stayonhire/feedback#94] [Feedback / Issues]
Hi gents,

I have been having some issues reloading the page when the computer/laptop has been sitting overnight. I have to go back to the stayonhire page and then re login each time.

Not a huge issue but I thought you should be aware.

**Comment by dylanhru** [comment]
Hi Wade,

Very interesting. There is a 'timed out/refresh' pop up that should appear after ~1 hour of idle time on a computer.

Have you ever seen this appear?

I will take a look into whether that has stopped performing its job, as the pop up should trigger a page refresh (to keep real-time updates synced) -but this shouldn't log you out of your account.

Just to confirm your problem, do you need to login again in the morning or do a page refresh?

**Comment by dylanhru** [comment]
Hi Wade, 

Do you ever see a screen/pop-up like this? The below screenshot is currently how idle time appears in the app.

![Screenshot 2023-12-21 at 3 21 01 pm](https://github.com/stayonhire/feedback/assets/34927957/f2b9dd45-790a-4f40-84c7-1a7b40eeb2b0)

If you have never seen this popup, then we may have an additional problem to look into?

**Comment by WadeRolfe** [comment]
Yep, that comes at times but that’s not the issue I have. I cant catch it at the moment but when I do I will screenshot and send it through.


Kind regards,

Wade Rolfe | Power Projects

M| 0408 288 082 E| ***@***.******@***.***> W| www.abpower.com.au<http://www.abpower.com.au/>

[Graphical user interface, website  Description automatically generated]

P Please consider the environment before you print this email

Legal Notice:
This transmission is intended for the person(s) named above. If you are not the person(s) then you are not entitled to the information. We do not waive any privilege or rights of confidentiality attaching to this transmission and any disclosure or use of this transmission by you is strictly prohibited. If you receive this transmission in error please notify the sender immediately. We accept no responsibility for any viruses that may be transmitted and recommend that you perform virus checks on this email and any attachments
